### DISCUSSION TITLE
How to set a keymap to insert an empty `python code block`?

### DISCUSSION THREAD

**Discussion by hopezh** [quarto-dev/quarto-nvim#147] [Q&A]
I'd like to ask how to config a keymap, e.g. `<leader>op`, to insert an empty `python code block` in a `qmd` file. 

(I forgot how I set it a while ago...)

**Comment by jmbuhr** [comment]
I have some that also automatically split a chunk if the cursor currently already is in one:
https://github.com/jmbuhr/quarto-nvim-kickstarter/blob/6454aea43ffd613aa525e097d0d0cd099b7b3aad/lua/config/keymap.lua#L162
